video/getting-started/setup-local-media/setup-local-media.swift

```diff
@@ -0,0 +1,18 @@
+// Create a local media object
+var localMedia = TVILocalMedia()
+
+// Add an audio track
+var enable = true;
+var localAudioTrack = localMedia?.addAudioTrack(enable)
+
+// Create camera object
+var camera = TVICameraCapturer()
+
+// Add a video track
+var localVideoTrack = localMedia?.addVideoTrack(enable, capturer: camera!)
```

Comment:
`camera` is also not an optional here so force unwrapping it is not necessary here.